auto_nag/scripts/missing_beta_status.py

```diff
@@ -32,29 +32,28 @@ def get_autofix_change(self):
 
     def handle_bug(self, bug, data):
         bugid = str(bug["id"])
-        central = bug[self.status_central]
+        nightly = bug[self.status_nightly]
         release = bug[self.status_release]
         doc = self.get_documentation()
 
-        if central == release:
-            if release != "verified":
+        # If the two status are different, we don't know what to set.
+        # If this affected on nightly and release, beta will be affected.
+        # Otherwise we cannot say per say if beta should be the same as nightly
+        # and release.
```

Comment:
And add: `, and we better be conservative in this case to avoid bugs falling through the cracks.`